UnitTests/ScientistTests.cs

```diff
@@ -7,17 +8,19 @@
 using UnitTests;
 using Xunit;
 
+using System.Reactive.Linq;
+
 public class TheScientistClass
 {
     public class TheScienceMethod
     {
         [Fact]
-        public void RunsBothBranchesOfTheExperimentAndReportsSuccess()
+        public async void RunsBothBranchesOfTheExperimentAndReportsSuccess()
```

Comment:
Should this return a `Task` here?  I know the article you had linked suggested that some testing frameworks may actually ignore this component.